src/mSupplyMobileApp.js

```diff
@@ -135,7 +135,6 @@ class MSupplyMobileAppContainer extends React.Component {
   };
 
   runWithLoadingIndicator = async functionToRun => {
-    UIDatabase.isLoading = true;
```

Comment:
these are just not set. Can check `UIDatabase` or `react-native-database` .. it's never used anywhere... and so I couldn't help myself 🤷‍♂